src/common/src/array/bool_array.rs

```diff
@@ -228,7 +228,9 @@ mod tests {
             let array = helper_test_builder(v.clone());
 
             let encoded = array.to_protobuf();
-            let decoded = read_bool_array(&encoded, num_bits).unwrap().into_bool();
+            let decoded = ArrayImpl::from_protobuf(&encoded, num_bits)
+                .unwrap()
+                .into_bool();
```

Comment:
Do we make private `read_bool_array` (or whole `column_proto_readers`)?

Reply:
Yes, I made the whole module private as I found there's no usages except for this one.